week-3/F-array-forEach/exercise.js

```diff
@@ -6,9 +6,47 @@
 
   An array with numbers 0-15 has been provided.
 */
+//create a function that performs on each element in the array.
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(multiples(item){ 
+if (item %3 === 0){
+  return "Fizz";
+} else if (item %5 === 0){
+  return "Buzz";
+} else if (item %3 === 0 && item %5 === 0){
+  return "FizzBuzz";
+} else {
+  return item;
```

Comment:
I think your checks are good here, but here's a couple of points:
- you don't have to `return` anything - you can just print it to the console
- I suggest starting with the "strictest" of the checks - for example with the number `15` it's meant to print "FizzBuzz". But if it encounters the `item % 3 === 0` check first, it'll print "Fizz". So the checks just need a reshuffle :)